src/main/java/no/nav/veilarbperson/client/pdl/domain/Familiemedlem.java

```diff
@@ -17,5 +17,6 @@
     LocalDate fodselsdato;
     String kjonn;
     LocalDate dodsdato;
+    String gradering;       //diskresjonskode
```

Comment:
Hvis det står beskrevet alle verdier som dette feltet kan ha i PDL, så er det kanskje bedre med ENUM

Reply:
Enig i at det bør være enum. Bør kanskje feile hvis det kommer en ukjent verdi?

Reply:
Jeg tror ikke vi har behov for å sjekke en ukjent verdi fordi vi er kun interessert i disse bestemt 3 graderinger (kode6/7/19).